### DISCUSSION TITLE
Multiple speakers

### DISCUSSION THREAD

**Discussion by specpro30** [neonbjb/tortoise-tts#162] [Q&A]
Hi there, thanks for sharing this awesome repo. The results are really something! Just a question regarding multiple speakers. Is there a way to define how many speakers there are and which voices are used for which speakers? Thanks!

**Comment by neonbjb** [comment]
I don't really understand what you mean. There should only be one speaker at a time. Tortoise does occasionally render multiple voices in a speech segment due to some problems I had with my dataset, but this should be considered a bug.

**Reply by MV799** [reply]
Noticed it too, and was a bit freaked out - how did I manage it :)
